lib/getComponentInstance/getComponentInstance.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+/**
+ * get component instance from uri
```

Comment:
Does this return the instance with `@published`? More explicit comment, por favor.